presto-kafka/src/main/java/io/prestosql/plugin/kafka/PlainTextKafkaConsumerFactory.java

```diff
@@ -56,6 +58,27 @@ public Properties configure()
         properties.setProperty(VALUE_DESERIALIZER_CLASS_CONFIG, ByteArrayDeserializer.class.getName());
         properties.setProperty(RECEIVE_BUFFER_CONFIG, Long.toString(kafkaBufferSize.toBytes()));
         properties.setProperty(ENABLE_AUTO_COMMIT_CONFIG, Boolean.toString(false));
+        if (kafkaConfig.getSecurityProtocol() != null) {
```

Comment:
Can we export them as a `Properties` from `KafakSecurityConfig` like we build `OrcReaderOptions` from `OrcReaderConfig` ?

Reply:
will expose the set of security related configs from KafkaSecurityConfig class...